client/listling/index.js

```diff
@@ -156,6 +156,8 @@ listling.ListPage = class extends micro.Page {
                 this._data.creatingItem = true;
                 this.querySelector(".listling-list-create-item [is=listling-item]").focus();
                 this.querySelector(".listling-list-create-item [is=listling-item]").scrollIntoView(false);
+                const elem = this.querySelector(".listling-list-create-item [is=listling-item]");
+                elem.startCreate(this._form.elements.item_template.value);
```

Comment:
I think you want to directly get the data from the model here, right? `elem.startCreate(this._data.lst.item_template)`